docs/source/cli/model-training-inference/real-time-inference.rst

```diff
@@ -0,0 +1,420 @@
+
+``graph`` (**Required**)
+>>>>>>>>>>>>>>>>>>>>>>>>>
+
+This ``graph`` field is similar to fields, e.g., ``graph`` field, in :ref:`GSProcessing input specification
+<gsprocessing_input_configuration>`. It contains three sub-fields, i.e., ``nodes``, ``edges``, and ``targets``.
+
+A ``nodes`` field contains a list of ``node`` fileds. A ``node`` includes the raw input data values
+of a node in the subgraph. It has the following required attributes.
+
+* ``node_type``: string, the raw node type name in a graph. It should be same as these ``node_type`` defined in
+  :ref:`gconstruct JSON specification <gconstruction-json>` or the ``type`` values of ``nodes`` defined in 
+  :ref:`gsprocessing JSON specification <gsprocessing_input_configuration>`.
+* ``node_id``: the raw node identifier.
+* ``features``: a dictionary, whose key is a feature name, and its value is the value of the feature.
+  feaure names should be same as these ``feature_name`` defined in :ref:`gconstruct JSON specification
+  <gconstruction-json>`, or these ``name`` values of ``features`` fields defined in
+  :ref:`gsprocessing JSON specification <gsprocessing_input_configuration>`.
+
+An ``edges`` field contains a list of ``edge`` fields. An ``edge`` includes the raw input data values of an
+edge in the subgraph. It has the following required attributes.
+
+* ``edge_type``: list, the raw edge type name in the format of a list with three elements, which indicate
+  source node type, edge type, and destination edge type. It should be same as these ``relation`` fileds defined
+  in :ref:`gconstruct JSON specification <gconstruction-json>` or these ``type`` values of ``source``
+  ``relation``, and ``dest`` fileds defined in :ref:`gsprocessing JSON specification <gsprocessing_input_configuration>`.
+* ``src_node_id``: user defined node identifier for the source node.
+* ``dest_node_id``: user defined node identifier for the destination node.
+* ``features``: a dictionary, whose key is a feature name, and its key is value of the feature. 
+  feaure names should be same as these ``feature_name`` defined in :ref:`gconstruct JSON specification
+  <gconstruction-json>`, or these ``name`` values of ``features`` fields defined in
+  :ref:`gsprocessing JSON specification <gsprocessing_input_configuration>`.
+
+A ``targets`` field contains a list of target ``node`` or ``edge`` fileds depending on the value of ``gml_task``
+These ``node`` or ``edge`` fileds is same as ``node`` and ``edge`` above, but the features field is not
+required. And they should be in the ``nodes`` or ``edges`` list of a ``graph``.
+
+An example payload JSON object is like the following:
+
+.. code:: yaml
+
+    {
+        "version": "gs-realtime-v0.1",
+        "gml_task": "node_classification",
+        "graph": {
+            "nodes": [
+                {
+                    "node_type": "author",
+                    "features": {
+                        "feat": [
+                            0.011269339360296726,
```

Comment:
Better to have a feat example of categorical features.

Reply:
added two categorical feature examples